modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+= Using skip node annotation
+
+* Use skip-node annotation to prevent the `node-labeller` from reconciling a node.
+
++
+[source,terminal]
+----
+$ oc annotate node {node} node-labeller.kubevirt.io/skip-node=true
```

Comment:
is `{node}` a value that needs to be substituted? If so, it should be something like `<node_name>`, and the step should mention that you need to substitute that value.